extensions/interactions/GraphInput/directives/graph-input-rules.service.spec.ts

```diff
@@ -194,7 +182,7 @@ describe('Graph Input service', function() {
     });
   });
 
-  var undirectedEmptyGraph = function() {
+  let undirectedEmptyGraph = (): any => {
```

Comment:
Since this doesn't actually return an `any` do add the TODO to https://github.com/oppia/oppia/issues/7165

Reply:
Same for everywhere that any is used as a placeholder

Reply:
Done